src/components/site/LogoType.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import LogoType from "./LogoType";
+
+describe("Text Input", () => {
```

Comment:
you can remove this describe it says ("Text Input") but it's really LogoType. However we know what this test covers by the name of the file.

Reply:
removed